test/input/beamville/beam.conf

```diff
@@ -283,9 +287,10 @@ matsim.modules {
   }
   controler {
     firstIteration = 0
-    lastIteration = 10
+    lastIteration = 1
```

Comment:
Was this a remnant of testing? Should it be changed for pushing in?

Reply:
Hi @JustinPihony, yes it was remnant of testing. I have reverted this. I will make sure this should not happen again